client/src/components/main/Favorite/AddShopToFavorite.js

```diff
@@ -85,4 +101,4 @@ const AddShopToFavorite = props => {
   );
 };
 
-export default AddShopToFavorite;
+export default withRouter(AddShopToFavorite);
```

Comment:
Since this is a functional component not a class component it makes much more sense here to use a hook. It seems like you're trying to get access to the url params, although I'm not 100% sure, if it is the params you're after I'd recommend using the following at the very top of the component `const { shopId } = useParams();` instead of using `withRouter` which hooks into a bunch of additional stuff you don't need

Reply:
I added thinking that I will fetch the userId from params, then forgot to delete it.